app/poro/tour.rb

```diff
@@ -3,4 +3,16 @@ class Tour
   def navigation_link(coordinates)
     NavigationService.new.get_tour(coordinates)
   end
+
+  def arrange_coordinates_by_latitude(coordinates)
+    coordinates.sort do |a, b|
+      a[0] <=> b[0]
+    end
+  end
+
+  def arrange_coordinates_by_longitude(coordinates)
+    coordinates.sort do |a, b|
+      a[1] <=> b[1]
+    end
+  end
```

Comment:
These methods are indeed repetitive. They are simple so I didn't think it was worth trying to make one polymorphic method that took a lat or lon argument. If you have any suggestions, I'm happy to adjust!

Reply:
i've implemented polymorphism for 2-3 methods before. not sure when "worth it" comes into play, i'm still new at this